include/freerdp/settings.h

```diff
@@ -1352,7 +1360,14 @@ extern "C"
 		ALIGN64 char* GatewayHttpExtAuthBearer;   /* 2002 */
 		ALIGN64 char* GatewayUrl;                 /* 2003 */
 		ALIGN64 BOOL GatewayArmTransport;         /* 2004 */
-		UINT64 padding2015[2015 - 2005];          /* 2005 */
+		ALIGN64 char* GatewayAvdWvd;              /* 2005 */
```

Comment:
Again - this could be better named

Reply:
ok, renamed to `GatewayAvdWvdEndpointPool`